doc/api/cli.md

```diff
@@ -181,6 +181,14 @@ Make built-in language features like `eval` and `new Function` that generate
 code from strings throw an exception instead. This does not affect the Node.js
 `vm` module.
 
+### `--dns-verbatim=value`
```

Comment:
I am sorry but I still prefer the `--dns-order=verbatim` or `--dns-order=ipv4` approach here and would.